src/Core/Checkout/Order/SalesChannel/CancelOrderRoute.php

```diff
@@ -36,18 +36,22 @@ public function getDecorated(): AbstractCancelOrderRoute
      * @Since("6.2.0.0")
      * @OA\Post(
      *      path="/order/state/cancel",
-     *      summary="Cancel a order",
+     *      summary="Cancel an order",
      *      operationId="cancelOrder",
      *      tags={"Store API", "Order"},
      *      @OA\RequestBody(
      *          required=true,
      *          @OA\JsonContent(
-     *              @OA\Property(property="orderId", description="The id of the order to be changed", type="string")
+     *              @OA\Property(
+     *                  property="orderId",
+     *                  description="The id of the order to be canceled",
+     *                  type="string"
+     *              )
      *          )
      *      ),
      *      @OA\Response(
      *          response="200",
-     *          description="State Machine",
+     *          description="State machine state",
```

Comment:
Which state machine? What do I do with it? Think from the view of an API consumer. What would they expect as a response, what would they do with that response? Maybe an example?